spec/features/staff/beis_users_can_edit_a_user_spec.rb

```diff
@@ -7,19 +7,33 @@
     stub_auth0_token_request
   end
 
+  scenario "the email address is disabled" do
+    user = create(:beis_user)
+    authenticate!(user: user)
+
+    target_user = create(:administrator, name: "Old Name", email: "old@example.com")
+
+    visit organisation_path(user.organisation)
+    click_on(t("page_title.users.index"))
+
+    find("tr", text: target_user.name).click_link("Edit")
+
+    email_field = find("input[name='user[email]']")
+    expect(email_field["disabled"]).to eq("disabled")
```

Comment:
Capybara provides a `disabled?` method on returned elements, so you can do:

`expect(email_field).to be_disabled`